docs/03_server/05_access-control/03_password_authentication.md

```diff
@@ -530,51 +631,78 @@ Note the following:
     DETAILS.NAME = SALES_TRADERS
 
 #### Delete response
-    MESSAGE_TYPE = EVENT_DELETE_PROFILE_ACK
+    MESSAGE_TYPE = EVENT_ACK
 
 ### Insert User
 
 #### Insert request
     MESSAGE_TYPE = EVENT_INSERT_USER
-    USER_NAME = jwolf
+    USER_NAME = JohnDoe
     DETAILS.USER_NAME = JohnWolf
     DETAILS.FIRST_NAME = John
     DETAILS.LAST_NAME = Wolf
     DETAILS.EMAIL_ADDRESS = john.wolf@genesis.global
     DETAILS.STATUS = ENABLED
-    DETAILS.RIGHT[0].ID = 00000000000001RISP0
```

Comment:
IDs don't exist and we don't allow rights to be inserted for a specific user, only profiles.